core/src/Revolution/Processors/Security/Access/Policy/GetList.php

```diff
@@ -137,16 +134,35 @@ public function prepareRow(xPDOObject $object)
                     }
                 }
             }
-            $policyArray['active_permissions'] = $ct;
-            $policyArray['active_of'] = $this->modx->lexicon('active_of', [
-                'active' => $policyArray['active_permissions'],
-                'total' => $policyArray['total_permissions'],
+            $policy['active_permissions'] = $ct;
+            $policy['active_of'] = $this->modx->lexicon('active_of', [
+                'active' => $policy['active_permissions'],
+                'total' => $policy['total_permissions'],
             ]);
-            $policyArray['permissions'] = $permissions;
+            $policy['permissions'] = $permissions;
         }
 
-        unset($policyArray['data']);
+        unset($policy['data']);
+
+        $policy['description_trans'] = $this->modx->lexicon($policy['description']);
+
+        return $policy;
+    }
+
+    /**
+     * @param xPDOObject|modAccessPolicy $object
+     *
+     * @return string
+     */
+    protected function prepareRowClasses(xPDOObject $object)
+    {
+        if ($object->isCorePolicy($object->get('name'))) {
```

Comment:
Shouldn't this be a `if not isCorePolicy` check?

Reply:
Yep, you are right. I fixed it. Also in templates as well.